netdata.spec.in

```diff
@@ -47,18 +47,25 @@ make %{?_smp_mflags}
 rm -rf "%{buildroot}"
 make %{?_smp_mflags} install DESTDIR="%{buildroot}"
 find "%{buildroot}" -name .keep -exec rm {} \;
+touch %{buildroot}/%{_sysconfdir}/netdata/netdata.conf
```

Comment:
My preference for packages is that they will run with a default config out of the box.  As it currently builds, netdata won't start because there is no config file.  There isnt' even a default config in the release tarball at all, so I just create an empty file and let it start up with the defaults.